src/component/App/App.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-import './App.module.css';
+import styles from './App.module.css';
 import ButtonPanel from '../buttonPanel';
 import Display from '../Display';
 
 function App() {
   return (
-    <React.Fragment className="App">
-      <Display />
-      <ButtonPanel />
-    </React.Fragment>
+    <>
+      <div className={styles.App}>
+        <Display />
+        <ButtonPanel />
+      </div>
+    </>
```

Comment:
- [ ] [REQUIRED] What you have done here is similar to:
```JS
<div className={styles.App}>
        <Display />
        <ButtonPanel />
 </div>
```
What you have to do is:
```JS
<>
   <Display />
   <ButtonPanel />
</>
```

Don't pay attention to the style in this milestone. You will work on styling in `milestone 3`

Reply:
so you want me to remove it now is it written somewhere that If something is styled you have to remove it

Reply:
Its only aligning the items

Reply:
If it is styled is it making any logical errors